app/views/teachers/ConvertToTeacherAccountView.coffee

```diff
@@ -0,0 +1,138 @@
+RootView = require 'views/core/RootView'
+forms = require 'core/forms'
+TrialRequest = require 'models/TrialRequest'
+TrialRequests = require 'collections/TrialRequests'
+AuthModal = require 'views/core/AuthModal'
+storage = require 'core/storage'
+errors = require 'core/errors'
+User = require 'models/User'
+ConfirmModal = require 'views/editor/modal/ConfirmModal'
+
+FORM_KEY = 'request-quote-form'
```

Comment:
What do you mean by "playing"? Do you mean as in like in an experiment? It's not especially experimental. Local storage is being used to store the contents of the form whenever it changes, and filling in the stored values if the page is reloaded or navigated away from and returned to.